### ISSUE TITLE
Extend the user-defined rules language

### ISSUE THREAD

**Issue by deemp** [objectionary/normalizer#45]
Depends on #11.

Features:
- [ ] several patterns in conditions to match
- [ ] combine objects, resolve naming conflicts
- [ ] access attributes after renaming

## Several patterns in conditions to match

Example:

```yaml
TODO
```

## Several patterns in conditions to match

Example:

```yaml
TODO
```

## Access attributes after renaming

Example:

```yaml
TODO
```

**Comment by deemp** [comment]
Superseded by https://github.com/objectionary/normalizer/issues/51
